### ISSUE TITLE
LC0003 Do not use an Object ID for properties - False positive on AccessByPermission = TableData

### ISSUE THREAD

**Issue by rvanbekkum** [StefanMaron/BusinessCentral.LinterCop#53]
Hi @StefanMaron,
It seems like rule LC0003 gives a false positive on the `AccessByPermission` for the following sample:
```al
action("Reservation Entries")
{
    AccessByPermission = TableData Item = R;
    ApplicationArea = Reservation;
    Caption = 'Reservation Entries';
    Image = ReservationLedger;
    ToolTip = 'View all reservations that are made for the item, either manually or automatically.';

    trigger OnAction()
    begin
        Rec.ShowReservationEntries(true);
    end;
}
```

![sample_LC0003_false_positive](https://user-images.githubusercontent.com/7383241/135634098-0354d385-9fd6-45ea-ab85-1d1d47ff6e4b.png)

Could you have a look?

Thank you for the code analyzer and extension! :)

**Comment by rvanbekkum** [comment]
LC0005 raises a warning at the same time, which also seems to be a false positive.

The warnings raised on this line:
```
Do not use an Object ID for properties or variables declaration. Use TableData Item = R instead.AL(LC0003)
Wrong casing detected! Use TableData Item = R instead.AL(LC0005)
```
